wurzel/steps/qdrant/step.py

```diff
@@ -103,6 +104,7 @@ def _insert_embeddings(self, data: DataFrame[EmbeddingResult]):
                 payload={
                     "url": row["url"],
                     "text": row["text"],
+                    "text_tlsh_hash": tlsh.hash(row["text"].encode("utf-8")),
```

Comment:
multivector change missing

Reply:
Okay, should we first fix the typo in `step_mutli_vector.py`?

Reply:
yea good hint, can be part of this PR/MR